__tests__/app.test.js

```diff
@@ -454,3 +454,47 @@ test("POST: 404 with Not Found for non-existent article ID", () => {
       expect(response.body.msg).toBe("Bad Request")
     })
 })
+// 8==========
+test("should update an artcle votes with a positive value", (done) => {
+  request(app)
+    .patch("/api/articles/1")
+    .send({ inc_votes: 5 })
+    .expect(200)
+    .end((err, res) => {
+      if (err) return done(err)
```

Comment:
I am not at all familiar with this .end Is it required for your testing to work? If so fine but It's not something we teach on the course.